demos/src/scss/_demos.scss

```diff
@@ -48,10 +48,9 @@ body:before {
 }
 
 .o-grid-container {
-	background: lighten(#f6e9d8, 8.5%);
+	background: #EAE8DF;
 }
```

Comment:
The current architecture means that the demo is non-symmetrical, which looks like a bug:

![image](https://cloud.githubusercontent.com/assets/1735391/15596532/36d39c7c-2403-11e6-881c-6da1ba88ae22.png)

Note that the row appears to touch the grid container on the left, but not on the right.  This is in fact the way the grid is currently designed, so isn't a bug, but looks like one.  I thought it was therefore misleading to colour the row backgrounds, which is why I removed that below.  Changing the one above was more just because on my screen it is too light to be discernable, so I can't see it.